code/modules/surgery/bodyparts/head.dm

```diff
@@ -247,6 +247,7 @@
 				if(sprite_accessory)
 					//Create the overlay
 					facial_overlay = mutable_appearance(sprite_accessory.icon, sprite_accessory.icon_state, -HAIR_LAYER)
+					facial_overlay.overlays += emissive_blocker(facial_overlay.icon, facial_overlay.icon_state, alpha = facial_overlay.alpha)
```

Comment:
IIRC hair overlays actually get their alpha set later in a proc chain, so use the `hair_alpha` var. I think.